<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PHE</string>
  </property>
  <property name="toolTipDuration">
   <number>-5</number>
  </property>
  <property name="styleSheet">
   <string notr="true">.header {
	font-size: 12pt;
	color: rgba(0,0,0,200);
}

QWidget {
	font-family: &quot;Segoe UI&quot;, sans-serif;
	font-size: 9pt;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_cameraImage">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHuib">
    <property name="title">
     <string>Hub</string>
    </property>
    <addaction name="actionConnectHub"/>
    <addaction name="actionDisconnectHub"/>
    <addaction name="separator"/>
    <addaction name="actionChangeHub"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionConnectCamera"/>
    <addaction name="actionDisconnectCamera"/>
    <addaction name="separator"/>
    <addaction name="actionChangeCamera"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionUpdate_image"/>
    <addaction name="separator"/>
    <addaction name="actionRotate_90_clockwise"/>
    <addaction name="actionRotate_90_anti_clockwise"/>
    <addaction name="separator"/>
    <addaction name="actionFlip_horizontal"/>
    <addaction name="actionFlip_vertical"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionPerformance"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHuib"/>
   <addaction name="menuCamera"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionDisconnectHub">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionConnectHub">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionConnectCamera">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnectCamera">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionChangeHub">
   <property name="text">
    <string>Change</string>
   </property>
  </action>
  <action name="actionChangeCamera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionUpdate_image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Update image</string>
   </property>
  </action>
  <action name="actionRotate_90_clockwise">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate 90° clockwise</string>
   </property>
  </action>
  <action name="actionRotate_90_anti_clockwise">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate 90° anti-clockwise</string>
   </property>
  </action>
  <action name="actionFlip_horizontal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flip horizontal</string>
   </property>
  </action>
  <action name="actionFlip_vertical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flip vertical</string>
   </property>
  </action>
  <action name="actionPerformance">
   <property name="text">
    <string>Performance</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConnectCamera</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectToCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConnectHub</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectToBridge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDisconnectCamera</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>disconnectFromCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDisconnectHub</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>disconnectFromBridge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChangeHub</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectToNewBridge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChangeCamera</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectToNewCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpdate_image</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeImageUpdatePreference(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRotate_90_clockwise</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>rotateImageClockwise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRotate_90_anti_clockwise</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>rotateImageAntiClockwise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFlip_horizontal</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>flipImageHorizontal(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFlip_vertical</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>flipImageVertical(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOptions</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showOptions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPerformance</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showPerformance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectToNewBridge()</slot>
  <slot>disconnectFromBridge()</slot>
  <slot>connectToNewCamera()</slot>
  <slot>disconnectFromCamera()</slot>
  <slot>connectToBridge()</slot>
  <slot>connectToCamera()</slot>
  <slot>changeImageUpdatePreference(bool)</slot>
  <slot>rotateImageAntiClockwise()</slot>
  <slot>rotateImageClockwise()</slot>
  <slot>flipImageVertical(bool)</slot>
  <slot>flipImageHorizontal(bool)</slot>
  <slot>showOptions()</slot>
  <slot>showPerformance()</slot>
 </slots>
</ui>
